charts/signoz/templates/schema-migrator-async/migrations.yaml

```
@@ -1,35 +1,36 @@
 metadata:
-  name: {{ include "schemaMigrator.fullname" . }}-async
+  name: {{ include "schemaMigrator.fullname" . }}-async{{ $schemaMigratorSuffix }}
   labels:
     {{- include "schemaMigrator.selectorLabels" . | nindent 4 }}
-  {{- if .Values.schemaMigrator.annotations }}
   annotations:
-    {{- toYaml .Values.schemaMigrator.annotations | nindent 4 }}
-  {{- end }}
+    {{- with .Values.schemaMigrator.annotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

yamllint

[warning] wrong indentation: expected 2 but found 4

(indentation)